src/main/java/org/opentripplanner/netex/mapping/StopTimesMapper.java

```diff
@@ -128,6 +128,13 @@ StopTimesMapperResult mapToStopTimes(
 
       StopPointInJourneyPattern stopPoint = findStopPoint(pointInJourneyPattern, journeyPattern);
 
+      if (
+        stopPoint != null &&
+        (isFalse(stopPoint.isForAlighting()) && isFalse(stopPoint.isForBoarding()))
+      ) {
+        continue;
```

Comment:
Is there a reason to skip these? They are already marked as for not for alighting/boarding in `mapToStopTime`

Reply:
Our dataset contains many of these stops as they're usually used to model tariff barriers. There are no PassengerStopAssignments for these stopPoints, so "lookUpStopLocation" yields null which results in the trip not being mapped at all.
The attempt was to ignore these for now, but I can imagine better solutions

Reply:
Our dataset contains many of these stops as they're usually used to model tariff barriers. There are no PassengerStopAssignments for these stopPoints, so "lookUpStopLocation" yields null which results in the trip not being mapped at all.
The attempt was to ignore these for now, but I can imagine better solutions